### PR TITLE
Add Payment Handler API announcement article

### PR DESCRIPTION
**Pull request by agektmr**
What's changed, or what was fixed?
- Added Payment Handler API announcement article
- Added generic featured image for payment (credit_card.png)
- Added wf_blink_components to an existing payment request article

**Target Live Date:** 2018-06-07

- [ ] This has been reviewed and approved by @zkoch @dgash
- [x] I have run `gulp test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

This pull request is to stage the update and get stakeholders review the content.

**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2018/06/payment-handler-api.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+The Payment Handler API opens up a whole new ecosystem to the world. It allows a 
+website to act as a payment method and to be integrated into merchant websites 
+through the standard Payment Request API.
+
+## User experience
+
+From the end user's point of view, the user experience would look like this:
+
+<img src="/web/updates/images/2018/06/payment-handler-api.gif" style="float:right"/>
```
Comment (petele) [inline src/content/en/updates/2018/06/payment-handler-api.md]: You can use `class="attempt-right"` for floating images on the right.

File: src/content/en/updates/2018/06/payment-handler-api.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+1. [A payment method 
+   identifier](https://w3c.github.io/payment-method-id/#dfn-payment-method-identifiers): 
+   points to a payment method manifest.
+1. [A payment method manifest](https://w3c.github.io/payment-method-manifest/): 
+   points to a web app manifest and supported origins.
+1. [A web app manifest](https://developer.mozilla.org/en-US/docs/Web/Manifest): 
+   describes a website that hosts a service worker that handles payment 
+   requests.
+
+[To learn more about how to implement a payment app, read this
```
Comment (petele) [inline src/content/en/updates/2018/06/payment-handler-api.md]: Should be: "To learn more, read [Quick guide to implementing a payment app with the Payment Handler API](link)"

The link title should be the title of the document that you want them to read. Makes skimming easier and improves search-ability of docs.

File: src/content/en/updates/2018/06/payment-handler-api.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+kind of payment method, supported methods can include:
+
+* Bank transfers
+* Cryptocurrencies
+* E-money
+* Carrier billings
+* Merchant's point system
+* Cash on delivery (Merchant's self-served)
+
+## link to
```
Comment (petele) [inline src/content/en/updates/2018/06/payment-handler-api.md]: Link to?

File: src/content/en/updates/2018/06/payment-handler-api.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+
+* demo (bobpay)
+* [document](https://docs.google.com/document/d/1wM9b3szNH4-w0tpIefjLYSGNtyjLr31Q4ARNTB52bJ0/edit#)
```
Comment (petele) [inline src/content/en/updates/2018/06/payment-handler-api.md]: Should be more descriptive.

**Comment by jpmedley** [comment]
Any chance of getting this out before 1:00 PM, PDT? @agektmr if you can, I won't need to remove the link from the beta blog post.

@dgash @petele

**Comment by agektmr** [comment]
Replaced heavy gif image with a video uploaded to YouTube.

**Pull request merged**